pages/linux/lastb.md

```diff
@@ -0,0 +1,23 @@
+# lastb
+
+> Show a listing of last logged in users.
+
+- Show a list of all last logged in users:
```

Comment:
This commands needs sudo to run. Let us add that.

Reply:
You are right. But not everyone has `sudo` installed, for someone it might would be `su -c 'lastb'`.
What about beginning the line with a `#` to indicate that this command must be run as root?

Which of these suggestions do you prefer? Still `sudo`?